.circleci/config.yml

```diff
@@ -16,16 +16,14 @@ jobs:
       # Download and cache dependencies
       - restore_cache:
           keys:
-          - v1-dependencies-{{ checksum "yarn.lock" }}
-          # fallback to using the latest cache if no exact match is found
-          - v1-dependencies-
+          - v2-dependencies-{{ checksum "yarn.lock" }}
```

Comment:
@cansin, I couldn't get CI to pass without changing this in order to clear the cache :-/

Reply:
Huh that is weird, given you update yarn.lock I’d have assumed cache to invalidate. In any case, this changeset is fine so carry on.